.gitattributes

```diff
@@ -1,2 +1,4 @@
 # A hint to GitHub to show autogenerated files (*.gen.go) less prominently in diffs
 *.gen.go linguist-generated=true
+tests/e2e_regression/rpc-cache/**/*.pix filter=lfs diff=lfs merge=lfs -text
```

Comment:
:shrug: I don't see a big benefit either way. This felt a tiny bit more future proof in case we store any "normal" files under `rpc-cache/`, e.g. a README.

Reply:
oh we only need the .psg files right? pogreb scans them and generates the rest. could save time to deliver these pix files. what's pmt?